src/modules/battery_status/analog_battery.h

```diff
@@ -96,5 +97,17 @@ class AnalogBattery : public Battery
 	uORB::Subscription _vehicle_status_sub{ORB_ID(vehicle_status)};
 	uint8_t _arming_state{0};
 
+#if defined(BOARD_BATTERY_ADC_VOLTAGE_FILTER_S) || defined(BOARD_BATTERY_ADC_CURRENT_FILTER_S)
+	hrt_abstime _last_timestamp {0};
+#endif
+
+#ifdef BOARD_BATTERY_ADC_VOLTAGE_FILTER_S
+	AlphaFilter<float> _voltage_filter;
```

Comment:
```suggestion
	AlphaFilter<float> _voltage_filter{BOARD_BATTERY_ADC_VOLTAGE_FILTER_S};
```
You can now also just set the time constant as such, so you don't need to set it later and just need to provide `dt` when calling the update function.

Reply:
Thanks I'll update this.